_posts/2018-04-21-welcome-to-this-week-in-rust.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "Welcome to this week in Rust-WASM! 001"
+date:   2018-04-21 21:15:00 +0200
+---
+Hello and welcome to the another issue of *This Week in Rust+WASM*!
+
+[Rust](http://rust-lang.org) is a systems language pursuing the trifecta: safety, concurrency, and speed.
+
+[WASM](http://webassembly.org) is designed as a portable target for compilation of high-level languages like C/C++/Rust, enabling deployment on the web for client and server applications.
```

Comment:
Personally, I like referring to "WebAssembly" in writing instead of "wasm" or "WASM". I use `.wasm` for referring to the binaries themselves.